azure-pipelines/ui-automation/msal-broker-BrokerHost-nightly.yml

```diff
@@ -183,7 +183,7 @@ stages:
                   --use-orchestrator \
                   --environment-variables "clearPackageData=true" \
                   --results-history-name "$(resultsHistoryName)" \
-                  --test-targets "package com.microsoft.identity.client.msal.automationapp.testpass.broker"
+                  --test-targets "package com.microsoft.identity.client.msal.automationapp.testpass.broker.brokerapi, package com.microsoft.identity.client.msal.automationapp.testpass.broker.atpop, package com.microsoft.identity.client.msal.automationapp.testpass.broker.crosscloud, package com.microsoft.identity.client.msal.automationapp.testpass.broker.flw"
```

Comment:
Is this actually needed? I think just having `com.microsoft.identity.client.msal.automationapp.testpass.broker` should've been good enough because the rest are just sub-packages.

If that is really not working then you may want to try:
`com.microsoft.identity.client.msal.automationapp.testpass.broker.*` and see if that works

Reply:
This was added so that we don't attempt to run every test case and run into the 45m limit. The packages included here contain about half the test cases. I ended up adding a parameter to feed this through.